src/combobox/combobox.component.ts

```diff
@@ -286,12 +286,15 @@ export class ComboBox implements OnChanges, OnInit, AfterViewInit, AfterContentI
 	hostkeys(ev: KeyboardEvent) {
 		if (ev.key === "Escape") {
 			this.closeDropdown();
-		} else if (ev.key === "ArrowDown" && !this.dropdownMenu.nativeElement.contains(ev.target)) {
+		} else if (ev.key === "ArrowDown"
```

Comment:
This might be a good time to add IE11 specific key values
```suggestion
		} else if ((ev.key === "ArrowDown" || ev.key === "Down") // `"Down"` is IE specific value
```

Reply:
🙌 absolutely